### ISSUE TITLE
cuDNN Error: CUDNN_STATUS_BAD_PARAM

### ISSUE THREAD

**Issue by legend-k** [pjreddie/darknet#1428]
cuDNN status Error in: file: ./src/convolutional_layer.c : () : line: 293 : build time: Feb 13 2019 - 13:54:47 

- I am getting this error after pulling the latest commit.

- I don't get this error while simple running the darknet script, i only get this error when using the libdarknet.so file

- GPU=1
  CUDNN=1
  CUDNN_HALF=0
  OPENCV=1
  AVX=0
  OPENMP=0
  LIBSO=1

- CUDNN Ver: 7.0.5

- CUDA Ver: 9.0

**Comment by AlexeyAB** [comment]
> i only get this error when using the libdarknet.so file

How do you use it?
What cfg-file do you use?

**Comment by legend-k** [comment]
I am just modifying the darknet_video.py file to point to my weights, cfg, data files

This is my cfg(Tiny-Yolo-v3)
[net]
#Testing
batch=1
subdivisions=1
Training
#batch=32
#subdivisions=8
width=416
height=416
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.001
burn_in=1000
max_batches = 50000#500200
policy=steps
steps=400000,450000
scales=.1,.1

[convolutional]
batch_normalize=1
filters=16
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=32
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=64
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=128
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=1

[convolutional]
batch_normalize=1
filters=1024
size=3
stride=1
pad=1
activation=leaky

###########

[convolutional]
batch_normalize=1
filters=256
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[convolutional]
size=1
stride=1
pad=1
filters=18
activation=linear



[yolo]
mask = 3,4,5
anchors =  33, 24,  62, 39,  90, 66, 146,125, 284,218, 354,322
classes=1
num=6
jitter=.3
ignore_thresh = .7
truth_thresh = 1
random=1

[route]
layers = -4

[convolutional]
batch_normalize=1
filters=128
size=1
stride=1
pad=1
activation=leaky

[upsample]
stride=2

[route]
layers = -1, 8

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[convolutional]
size=1
stride=1
pad=1
filters=18
activation=linear

[yolo]
mask = 0,1,2
anchors =  33, 24,  62, 39,  90, 66, 146,125, 284,218, 354,322
classes=1
num=6
jitter=.3
ignore_thresh = .7
truth_thresh = 1
random=1

**Comment by legend-k** [comment]
I updated to  CUDA 10.0 and CUDNN 7.4.2, that error got resolved. but now I am getting Segmentation fault (core dumped) error

**Comment by sudharavali** [comment]
@legend-k Hi, were you able to resolve the issue  ?

**Comment by Davidray053** [comment]
have anyone got the solutions? i'm facing the same problem in google colab.
